ports/unix/Makefile

```diff
@@ -242,7 +242,7 @@ freedos:
 
 # build an interpreter for coverage testing and do the testing
 coverage:
-	$(MAKE) V=2 \
+	$(MAKE) \
```

Comment:
check: this change is on purpose?

Reply:
Ya, it reduces the verbosity which is my preference.

Reply:
got it!